node/declaration.test.ts

```diff
@@ -16,12 +16,13 @@ test('In source config takes precedence over netlify.toml config', () => {
   ]
 
   const funcConfig = {
-    geolocation: { path: '/geo-isc', cache: 'manual' },
-    json: { path: '/json', cache: 'off' },
+    geolocation: { path: ['/geo-isc', '/*'], cache: 'manual' },
+    json: { path: ['/json'], cache: 'off' },
```

Comment:
Can we maybe leave the `path` for `json` as a string to cover that flow?

Reply:
✅